### PR TITLE
Create items resource

### PR DESCRIPTION
**Pull request by serpient**
Create items resource routes and seed data

### CODE REVIEW

File: lib/rebay_api/listings/item.ex [hunk 1]
```diff
@@ -0,0 +1,22 @@
+defmodule RebayApi.Listings.Item do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "items" do
+    field :category, :string
+    field :description, :string
+    field :end_date, :utc_datetime
+    field :image, :string
+    field :price, :float
```
Comment (zacholauson) [inline lib/rebay_api/listings/item.ex]: https://husobee.github.io/money/float/2016/09/23/never-use-floats-for-currency.html

**Comment by zacholauson** [comment]
Tests are lookin good!

**Comment by serpient** [comment]
> Tests are lookin good!

The tests were automatically generated!!

**Comment by zacholauson** [comment]
> > Tests are lookin good!
> 
> The tests were automatically generated!!

lol, that is wild

File: lib/rebay_api/listings/item.ex (hunk 1 above)
Comment (serpient) [reply lib/rebay_api/listings/item.ex]: Done! See latest commit

**Review by zacholauson — APPROVED** [review]

**Pull request merged**